pkg/localnet/helpers.go

```diff
@@ -78,6 +80,19 @@ func TrackSubnet(
 		perNodeBlockchainConfig,
 		wallet,
 	); err != nil {
+		if errors.Is(err, context.DeadlineExceeded) {
+			printFunc("")
+			printFunc("A context timeout has occurred while trying to bootstrap the blockchain.")
+			printFunc("")
+			logPaths, _ := GetTmpNetAvailableLogs(networkDir, blockchainID, false)
+			if len(logPaths) != 0 {
+				printFunc("Please check this log files for more information on the error cause:")
+				for _, logPath := range logPaths {
+					printFunc("  " + logPath)
```

Comment:
Can we add unit test for helpers_test.go where we intentionally cause error and verify that we get the expected logs dir

Reply:
added test for GetTmpNetAvailableLogs